src/components/Form/formFunctions.ts

```diff
@@ -17,6 +17,7 @@ export enum Errors {
     invalid = "Please enter a valid entry.",
     submit = "Please ensure all fields have been entered correctly. Required fields are labelled with an asterisk.",
     external = "Please try again later.",
+    pastDate = "Please enter a date in the future.",
```

Comment:
should we not be able to backdate things? if I forgot to update a parcel but it's now been collected, what should happen?

Reply:
So ironically this is how it was implemented before. I think it is better to not allow it when adding a parcel as I can't imagine someone forgetting to add a parcel completely many times, and I think the number of errors it would catch makes it worth it, but I think whoever is implementing the edit parcel functionality should have some possibility to enter a past date as there needs to be some way to do it.